tests/generic_config_updater/gu_utils.py

```diff
@@ -238,3 +238,31 @@ def rollback_or_reload(duthost, cp=DEFAULT_CHECKPOINT_NAME):
     if output['rc'] or "Config rolled back successfull" not in output['stdout']:
         config_reload(duthost)
         pytest.fail("config rollback failed. Restored by config_reload")
+
+def check_show_ip_intf(duthost, intf_name, expected_content_list, unexpected_content_list, is_ipv4=True):
+    """Check lo interface status by show command
+
+    Sample output:
+    admin@vlab-01:~$ show ip interfaces  | grep Vlan1000
+    Vlan1000                   192.168.0.1/21       up/up         N/A             N/A
+    admin@vlab-01:~$ show ipv6 interfaces | grep Vlan1000
+    Vlan1000                          fc02:1000::1/64                             up/up         N/A             N/A
+                                      fe80::5054:ff:feda:c6af%Vlan1000/64                       N/A             N/A
+    """
+    address_family = "ip" if is_ipv4 else "ipv6"
+    output = duthost.shell("show {} interfaces | grep {} || true".format(address_family, intf_name))
```

Comment:
>grep [](http://example.com/codeflow?start=49&length=4)

`grep` will match more lines than you expected. For example `eth` will match `eth0`.

Reply:
good catch